test_regress/t/t_trace_ub_misaligned_address.pl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2023 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+if (!$Self->have_coroutines) {
+    skip("No coroutine support");
+}
+else {
+    top_filename("t/t_trace_ub_misaligned_address.v");
+
+    compile(
+        verilator_flags2 => ["--binary --timing --trace",
+                             "-CFLAGS -fsanitize=address,undefined",
+                             "-LDFLAGS -fsanitize=address,undefined"],
+        verilator_make_cmake => 0,
+        verilator_make_gmake => 0,
+        make_main => 0,
+        );
+
+    execute(
+        check_finished => 1,
+        );
+
+    files_identical("$Self->{obj_dir}/vlt_sim.log", $Self->{golden_filename}, "logfile");
```

Comment:
Is the .out file adding some sort of coverage? If not perhaps remove this and the .out?

Reply:
The intent is for the test to fail if any additional messages (such as the runtime warning about undefined behavior) appear.  Is there a way of doing that without using a .out file?

Reply:
Thats OK to have then.